api/apiFunctions.ts

```diff
@@ -38,34 +40,53 @@ export async function registerAccount({
 }
 
 /**
- * Login to an existing account
- * @param props - The account data
- * @param props.email - The email of the user
- * @param props.password - The password of the user
- * @returns {Models.Session | Error} - The session object or an error
+ * Recover a User Password
+ * @param props - the props for the recover password function
+ * @param props.email - the email of the user
+ * @returns {Promise<IRecoveryToken>} - the recovery token
  */
-export async function loginAccount({
+export async function recoverPassword({
   email,
-  password,
-}: IAccountData): Promise<Models.Session | Error> {
+}: {
+  email: string;
+}): Promise<IRecoveryToken> {
+  const baseURL = getBaseURL();
+
   try {
-    return await account.createEmailPasswordSession(email, password);
+    const result = await account.createRecovery(
+      email,
+      `${baseURL}/account/recovery`,
+    );
+    return result;
   } catch (error) {
     console.error(error);
-    throw new Error('Error logging in user');
+    throw new Error('Error recovering password');
   }
 }
 
 /**
- * Logout the current user
- * @returns {object | Error} - The session object or an error
+ * Reset a User Password
+ * @param props - the props for the reset password function
+ * @param props.userId - the user id
+ * @param props.token - the recovery token
+ * @param props.password - the new password
+ * @returns {Promise<void>} - the reset password
  */
-export async function logoutAccount(): Promise<object | Error> {
+export async function resetPassword({
+  userId,
+  token,
+  password,
+}: {
+  userId: string;
+  token: string;
+  password: string;
+}): Promise<IRecoveryToken> {
   try {
-    return await account.deleteSession('current');
+    const result = await account.updateRecovery(userId, token, password);
+    return result;
   } catch (error) {
-    console.error(error);
-    throw new Error('Error logging out user');
+    console.error('Password reset error:', error);
```

Comment:
remove

Reply:
@shashilo remove what?  I can't tell what you want me to remove.

Reply:
✅ updated!